tiktoken-rs/src/tokenizer.rs

```diff
@@ -136,11 +136,16 @@ mod tests {
 
     #[test]
     fn test_get_tokenizer() {
+        assert_eq!(get_tokenizer("gpt-4-0125"), Some(Tokenizer::Cl100kBase));
```

Comment:
nit: For the test I would use 
```suggestion
        assert_eq!(get_tokenizer("gpt-4-0125-turbo"), Some(Tokenizer::Cl100kBase));
```

Reply:
Hey @zurawiki

In the OpenAI docs, there's no model called `gpt-4-0125-turbo`, just `gpt-4-0125-preview`. Based on the description available [here](https://platform.openai.com/docs/models/gpt-4-and-gpt-4-turbo), it's a turbo model, but they didn't add `turbo` to its name.

Just to confirm, then, should I add `turbo` to the test anyway?

Thank you for taking your time to review this!

Reply:
Thank for pointing that out. I meant to say let's test for `gpt-4-0125-preview`

Reply:
Done! :^) (TBF, you did it, I just committed lol)